content/en/docs/topics/plugins.md

```diff
@@ -162,7 +233,25 @@ There are some strategies for working with plugin commands:
   `--help`. Helm will use `usage` and `description` for `helm help` and `helm
   help myplugin`, but will not handle `helm myplugin --help`.
 
+### Testing a Local Plugin
+
+First you need to find your `HELM_PLUGINS` path to do it run the folowing command:
+
+``` bash
+helm env
+```
+
+Change your current directory to the director that `HELM_PLUGINS` is set to.
+
+Now you can add a symbolic link to your build out put of your plugin in this example
```

Comment:
```suggestion
Now you can add a symbolic link to your build output of your plugin in this example
```

Reply:
Good catch, I just moved this block but I should have checked it.